src/scss/styles.scss

```diff
@@ -11,10 +11,15 @@ header {
 }
 
 .form-control {
-  max-width: 600px;
+  max-width: 25rem;
```

Comment:
Can you make this a  variable? I can see you repeat it in line 19.

Reply:
I've made the variable, and added som whitespace.